### DISCUSSION TITLE
Thermia - Options for extra steering and insights

### DISCUSSION THREAD

**Discussion by nmachtel** [klejejs/ha-thermia-heat-pump-integration#111] [Q&A]
Hi all

First of all - thanks for the great work for this integration. I'm fairly new to HA and to the topic, so don't hesitate to point me into the right direction. I hope to be able to add some value too to the community later on! 

We have a Calibra Eco 12 pump at home, and the instructions from our supplier are quite clear : 
- If you have too much (solar) energy -> go into boost mode.
- If you are way above your capacity and want to lower -> enable EVU mode (see description, in dutch, below). 

I only find two switches - one for hot water and one for hot water boost. Boost seems the boost to me, but is disabling the hot water the same as putting the pump into off (as much as possible, because it will still respect your floor and house temperature)? Or do I miss a parameter/some instructions?

Secondly, I would like to have a good view on when the pump is running and when not. I had in mind to use the sensor Status Heat Operational Status for this, but it does not seem in line with the graphs I have in the Thermia portal itself.

Again, if questions are stupid or double -> please don't hesitate to just put me into the right direction.

Kind regards, and thanks again!
Niels

<img width="629" alt="image" src="https://github.com/user-attachments/assets/da6afe11-64dc-4763-8935-aa5c9dac4231">

**Comment by klejejs** [comment]
Hey, I can help you and add additional switches or sensors to the integration if I have more information about your heat pump.
Please create a new issue with a type feature on GitHub issues tab and follow the instructions there as that will help me to understand how your heat pump works. If it is possible, please also add the screenshots of the wanted functionalities from the Thermia Online website as that helps connect the functionality with the expected behavior.
This could also help me figure out how the hot water switch works for you.
